frame/dynamic-fee/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "pallet-dynamic-fee"
-version = "2.0.0"
+version = "2.0.1-dev"
```

Comment:
```suggestion
version = "3.0.0-dev"
```

We bumped a major version of `pallet-evm` which this depends, and it's a backward incompatible change.

Reply:
I thought although it depends on `pallet-evm`, and that includes breaking changes, none of those change how `pallet-dynamic-fee`'s public api is exposed or it's behaviour in any way, and thus resulted in a patch version. Changing now.